nautobot_golden_config/utilities/utils.py

```diff
@@ -8,3 +10,24 @@ def get_platform(platform):
     if not PLUGIN_CFG.get("platform_slug_map"):
         return platform
     return PLUGIN_CFG.get("platform_slug_map").get(platform, platform)
+
+
+def get_secret_value(secret_type, git_obj):
+    """Get value for a secret based on secret type and device.
+
+    Args:
+        secret_type (SecretsGroupSecretTypeChoices): Type of secret to check.
+        git_obj (extras.GitRepository): Nautobot git object.
+
+    Returns:
+        str: Secret value.
+    """
+    try:
+        value = git_obj.secrets_group.get_secret_value(
+            access_type=SecretsGroupAccessTypeChoices.TYPE_HTTP,
```

Comment:
Is this the only value it could be?

Reply:
Right now yes.
https://github.com/nautobot/nautobot/blob/main/nautobot/extras/datasources/git.py#L147